src/ContentEditableInput.js

```diff
@@ -0,0 +1,469 @@
+import { replaceRange } from "./changes";
+import { contains, range, removeChildrenAndAdd, selectInput } from "./util/dom";
+import { on, signalDOMEvent } from "./util/event";
+import { applyTextInput, copyableRanges, disableBrowserMagic, handlePaste, hiddenTextarea, lastCopied, setLastCopied } from "./input";
+import { operation, runInOp } from "./operations";
+import Pos from "./Pos";
+import { cmp, maxPos, minPos } from "./Pos";
```

Comment:
And that (awkward double imports, or alternatively adding `default: Pos` in this one) is why I don't like default exports very much.